tests/integration/deploy-operator.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
+# Copyright 2022 Red Hat, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+install-operator-k8s() { # installs istio-operator on kubernetes
+    local ROOT
+    ROOT="$(git rev-parse --show-toplevel)"
+    local TAG
+    TAG="${TAG:-$(git rev-parse HEAD)}"
+    local NS="${NS:-openshift-operators}"
+    # var name IMAGE is in kind provisioner
+    local OPERATOR_IMAGE="${OPERATOR_IMAGE:-localhost:5000/istio-operator-integ}:${TAG}"
+    local ISTIO_CNI_IMAGE_NAME="${ISTIO_CNI_IMAGE_NAME:-quay.io/maistra-dev/istio-cni-ubi8-integ:latest}"
```

Comment:
Sorry, missed this. We don't need to use a real/external registry, do we? We can use the kind one, no? We don't want to pollute the quay registry with those images.

Reply:
Hi Jonh, the cni and istiod images are from maistra/istio . They are not built in istio-operator repo. 
So there is no image pushed to local registry. When I need to test the operator with a CR, I still need those two images as a sample.

That's why I pushed two 2.3 community images to quay.io/maistra-dev . They are like testdata samples.

Reply:
don't we push these as part of the daily build now?

Reply:
@dgn the daily build is using a tag with date, 
https://quay.io/repository/maistra-dev/pilot-ubi8?tab=tags
e.g. "2.3-daily-2022-12-20" and I am not sure when a daily build has been pushed to quay or a previous daily build tag has been cleaned.

So I tried to keep a sample copy of the image and use a static tag like 
https://quay.io/repository/maistra-dev/pilot-ubi8-integ?tab=tags